infra/build/functions/fuzzbench.py

```diff
@@ -310,23 +309,54 @@ def get_extract_crashes_steps(fuzzing_engine, project, env_dict):
   }
   steps.append(download_libfuzzer_build_step)
 
-  crashes_dir = f'{GCB_WORKSPACE_DIR}/crashes/'
   extract_crashes_step = {
       'name':
           get_engine_project_image_name(fuzzing_engine, project),
       'args': [
           'bash', '-c', f'unzip {libfuzzer_build_dir}{build_filename} '
-          f'-d {libfuzzer_build_dir} && mkdir -p {crashes_dir} && '
+          f'-d {libfuzzer_build_dir} && mkdir -p {OOD_CRASHES_DIR} && '
           f'{libfuzzer_build_dir}{env_dict["FUZZ_TARGET"]} {OOD_OUTPUT_CORPUS_DIR} '
-          f'-runs=0 -artifact_prefix={crashes_dir}; '
-          f'echo "\nCrashes found by OOD:" && ls {crashes_dir} '
+          f'-runs=0 -artifact_prefix={OOD_CRASHES_DIR}/; '
+          f'echo "\nCrashes found by OOD:" && ls {OOD_CRASHES_DIR} '
       ],
   }
   steps.append(extract_crashes_step)
 
   return steps
 
 
+def get_upload_testcase_steps(project, env_dict):
+  """Returns the build steps to upload a testcase in the ClusterFuzz External
+  upload testcase endpoint."""
+  steps = []
+
+  access_token_file_path = f'{GCB_WORKSPACE_DIR}/at.txt'
+  get_access_token_step = {
+      'name':
+          'google/cloud-sdk',
+      'args': [
+          'bash', '-c',
+          f'gcloud auth print-access-token > {access_token_file_path}'
+      ]
```

Comment:
Not a big deal here, but storing the access token to disk isn't ideal from a security perspective (even if the runner is ephemeral and the token isn't logged). Just for future consideration: you could avoid the file altogether by piping the token directly into the Python script:

```bash
gcloud auth print-access-token | python3 upload_testcase.py <crashes_dir> <job_name> <target_name> -
```
and then have the script read the token from `stdin`.

Or alternatively, use `subprocess` in the `main` function of upload_testcase.py:
```python
token = subprocess.check_output(["gcloud", "auth", "print-access-token"]).decode().strip()
```